README.md

```diff
@@ -41,24 +41,36 @@ We have introduced this feature with [APIM CLI](https://github.com/wso2/product-
 
 ##### Deploying APIM in  K8S Cluster
 
-Kubernetes artifacts to deploy APIM and APIM analytics deployment are shipped with the this distribution.
+Kubernetes artifacts to deploy APIM and APIM analytics deployment are shipped with the this distribution. Below commands allow users to run API Manager default profile and API Manager Analytics profile in kubernetes runtime. Please note you will need to have enough menory and CPU to run below commands.
 
 Navigate to wso2am-k8s-crds/apim-operator/apim-deployment/api-manager
 
-- Deploy API Manager in Kubernetes Cluster
+- Deploy API Manager in Kubernetes Cluster. Following command will deploy WSO2 API Manager all in one profile in kubernetes. With following command API Manager default profile will start run on kubernetes runtime under WSO2 namespace. Please see below command and its output. 
 
 ```$xslt
-apimcli apply -f api-manager/k8s-artifacts
+>>apimcli apply -f k8s-artifacts
+namespace "wso2" created
+configmap "apim-conf" created
+deployment.apps "wso2apim" created
+service "wso2apim-service" created
 ```
-- Deploy APIM Analytics in Kubernetes Cluster
+  You can check the the details about running server by checking running pods or services in kubernetes runtime. You can execute following command you see how WSO2 API Manager deployed in kubernetes.
```

Comment:
You can check the details about running the server by checking running pods or services in Kubernetes runtime. You can execute the following command and see how WSO2 API Manager is deployed in Kubernetes